src/weight_windows.cpp

```diff
@@ -184,7 +184,7 @@ WeightWindows::WeightWindows(pugi::xml_node node)
       fmt::format("In weight window domain {} the number of spatial "
```

Comment:
Should we remove the additional word "spatial" so it just reads "the number of energy/spatial bins" ?

Reply:
Ohh yeah for sure. I think it could also be more clear which number is which. (The expected number vs. the size of the array passed in).

I'll make those updates shortly. Thanks!